ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1449,6 +1450,19 @@ ip_routing: < true | false >
 ipv6_unicast_routing: < true | false >
 ```
 
+#### Router General configuration
+
+```yaml
+router_general:
+  vrfs:
+    < destination-vrf >:
+      leaks:
```

Comment:
I would recommend updating leaks to `leak_routes` to match cli.

Reply:
Fixed in 7b15495